### PR TITLE
upd(android-studio): `2020.3.1.25` -> `2021.1.1.20`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

**Review by Henryws — APPROVED** [review]

**Comment by Henryws** [comment]
Also as a heads up, use conventional commits for commit messages

---

conventionalcommits.org/

**Comment by cat-master21** [comment]
Issues with installing so plan to fix.

**Comment by wizard-28** [comment]
Is this ready for review? Why it a draft still?

**Comment by cat-master21** [comment]
Android-studio does not install properly and if I were to update GitHub there would be a lot of commits. The current state on my end is it can't find $STUDIO_JDK

File: packages/android-studio/android-studio.pacscript [hunk 1]
```diff
@@ -2,33 +2,33 @@ name="android-studio"
...
 install() {
-    # Moving all android studio files to /opt
-    sudo cp -a bin lib jre plugins license LICENSE.txt build.txt product-info.json "$STOWDIR/$name/opt/$name"
+    # Moving android studio files to /opt
+    sudo cp -a bin lib jre plugins license LICENSE.txt build.txt product-info.json NOTICE.txt brokenPlugins.db icons.db classpath.txt "$STOWDIR/$name/opt/$name"
 
     # Creating a launcher in $PATH
-    sudo ln -s "$STOWDIR/$name/opt/$name/studio.sh" "$STOWDIR/$name/usr/bin/android-studio"
-
+    sudo ln -sf "$STOWDIR/$name/opt/$name/bin/studio.sh" "/usr/bin/android-studio"
```
Comment (Henryws) [inline packages/android-studio/android-studio.pacscript:20]:
```suggestion
    sudo ln -sf "$STOWDIR/$name/opt/$name/bin/studio.sh" "$STOWDIR/$name/usr/bin/android-studio"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/android-studio/android-studio.pacscript (hunk 1 above)
Comment (cat-master21) [reply packages/android-studio/android-studio.pacscript:20]: I tried this but I get the error: source is an absolute symlink, and fails to install. No matter what I try I get this error except for a hard link, but that doesn't point to JDK or other android-studio files.

Comment (Henryws) [reply packages/android-studio/android-studio.pacscript:20]: How about moving `studio.sh` to `android-studio`?

Comment (cat-master21) [reply packages/android-studio/android-studio.pacscript:20]: Same thing as hard-linking; can't find JDK or other android-studio files.

**Comment by cat-master21** [comment]
Still once more, same issue, Android-studio can't find JDK and other needed files.
Screenshot
:

**Comment by cat-master21** [comment]
Because there doesn't appear to be a solution linking files this PR will be closed.

**Pull request closed without merging**